client/me/account/main.jsx

```diff
@@ -627,7 +627,7 @@ const Account = createReactClass( {
 				>
 					{ this.state.submittingForm
 						? translate( 'Saving…' )
-						: translate( 'Save Account Settings' ) }
+						: translate( 'Save account settings' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.